packages/neon/neon/lib/src/widgets/unified_search_results.dart

```diff
@@ -0,0 +1,142 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:meta/meta.dart';
+import 'package:neon/l10n/localizations.dart';
+import 'package:neon/src/bloc/result.dart';
+import 'package:neon/src/bloc/result_builder.dart';
+import 'package:neon/src/blocs/accounts.dart';
+import 'package:neon/src/blocs/unified_search.dart';
+import 'package:neon/src/models/account.dart';
+import 'package:neon/src/widgets/cached_image.dart';
+import 'package:neon/src/widgets/exception.dart';
+import 'package:neon/src/widgets/image_wrapper.dart';
+import 'package:neon/src/widgets/linear_progress_indicator.dart';
+import 'package:neon/src/widgets/list_view.dart';
+import 'package:neon/src/widgets/server_icon.dart';
+import 'package:neon/src/widgets/user_avatar.dart';
+import 'package:nextcloud/nextcloud.dart';
+import 'package:provider/provider.dart';
+
+@internal
+class NeonUnifiedSearchResults extends StatelessWidget {
+  const NeonUnifiedSearchResults({
+    super.key,
+  });
+
+  @override
+  Widget build(final BuildContext context) {
+    final accountsBloc = Provider.of<AccountsBloc>(context, listen: false);
+    final bloc = accountsBloc.activeUnifiedSearchBloc;
+    return ResultBuilder.behaviorSubject(
+      stream: bloc.results,
+      builder: (final context, final results) => NeonListView(
+        items: results.data?.entries,
+        isLoading: results.isLoading,
+        error: results.error,
+        onRefresh: bloc.refresh,
+        builder: (final context, final snapshot) => AnimatedSize(
+          duration: const Duration(milliseconds: 100),
+          child: _buildProvider(
+            context,
+            accountsBloc,
+            bloc,
+            snapshot.key,
+            snapshot.value,
+          ),
+        ),
```

Comment:
This is to animate in the elements?

I'd say AnimatedListView please but let's keep it as a followup for #585

Reply:
I think we can leave it as is.

The NeonListView change will also require some other smaller resteuctures here that don't make sense to have now.

I don't think all NeonListViews need to be animated but in the PR you can see a separate `NeonListView.custom` constructor. 

Passing an AnimatedSliverList (or however it's called) into that should do the trick.